api4/command_test.go

```diff
@@ -171,23 +170,24 @@ func TestMoveCommand(t *testing.T) {
 	}
 
 	rcmd1, _ := th.App.CreateCommand(cmd1)
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
```

Comment:
Same suggestion here:

```suggestion
	th.TestForAllClients(t, func(t *testing.T, client *model.Client4) {
```

Reply:
@mgdelacroix  original client was th.SystemAdminClient so this should be TestForSystemAdminAndLocal